pkgs/applications/science/electronics/librepcb/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchgit, qtbase, qttools, qmake, mesa, openssl, zlib }:
+
+stdenv.mkDerivation rec {
+	name = "librepcb-unstable";
+	version = "2017-12-29";
+	src = fetchgit {
+		url = "git://github.com/LibrePCB/LibrePCB";
+		rev = "4efb06fa42755abc5e606da4669cc17e8de2f8c6";
+		sha256 = "0r33fm1djqpy0dzvnf5gv2dfh5nj2acaxb7w4cn8yxdgrazjf7ak";
+	};
+
+	enableParallelBuilding=true;
```

Comment:
That should be the default for all qmake based projects. See: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/qt-5/hooks/qmake-hook.sh#L13